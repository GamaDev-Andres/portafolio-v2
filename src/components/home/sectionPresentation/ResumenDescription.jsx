import { miDescripcion } from '../../../helpers/data'

const ResumenDescription = () => {
  return (
    <>
      <span className="uppercase font-medium text-xl text-fondos">
        Hola soy
      </span>
      <h2 className="my-4 uppercase font-bold text-3xl ">
        <span className="text-blue-400">andres</span> gama
      </h2>
      <p className="whitespace-pre-wrap text-fondos">
        Un
        <span className="text-blue-400 font-medium">
          {' Front-End Web Developer '}
        </span>
        {miDescripcion.presentacion}
      </p>
    </>
  )
}

export default ResumenDescription
