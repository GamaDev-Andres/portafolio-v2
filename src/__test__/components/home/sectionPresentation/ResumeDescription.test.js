/**
 * @jest-environment jsdom
 */
import Enzyme from 'enzyme'
import ResumenDescription from '../../../../components/home/sectionPresentation/ResumenDescription'

describe('probando <ResumeDescription/>', () => {
  const wrapper = Enzyme.mount(
    <ResumenDescription />
  )
  test('debe montarse correctamente', () => {
    expect(wrapper).toMatchSnapshot()
    expect(wrapper.find('span').at(0).text().trim()).toBe('Hola soy')
  })
})
