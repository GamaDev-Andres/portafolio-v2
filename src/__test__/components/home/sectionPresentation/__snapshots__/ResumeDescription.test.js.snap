// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`probando <ResumeDescription/> debe montarse correctamente 1`] = `
Array [
  <span
    className="uppercase font-medium text-xl text-fondos"
  >
    Hola soy
  </span>,
  <h2
    className="my-4 uppercase font-bold text-3xl "
  >
    <span
      className="text-blue-400"
    >
      andres
    </span>
     gama
  </h2>,
  <p
    className="whitespace-pre-wrap text-fondos"
  >
    Un
    <span
      className="text-blue-400 font-medium"
    >
       Front-End Web Developer 
    </span>
    apasionado por el mundo del desarrollo y por crear aplicaciones e interfaces que aporten y brinden grandes experiencias.
  </p>,
]
`;
